### ISSUE TITLE
Status

### ISSUE THREAD

**Issue by Dutchboy** [vadimpopa/ionic2-jspm-seed#4]
Hi!

My guess is that this repo is not maintained? Because, from what I see, is that:
- the referred versions of external libs are not up to date.
- there is almost no similarity to a default ionic2 structure, so it is impossible to merge it

What my best guess is, to start a sort of plugin, which can be merged into a default ionic structure

**Comment by vadimpopa** [comment]
Yes, I didn't have time to update it. Would you need jspm instead of default ionic cli with webpack ?

**Comment by Dutchboy** [comment]
I'm not a fan of Webpack and I've chosen jspm... 

Have you seen the Pluralsight jspm course, "Modern Modular Javascript With SystemJS and JSPM"?.. Get it, you'll love it... (I'll get it for you)

Tomorrow, what I'm aiming to construct is the following....
1. Make one (or two? or more) external gulp-task to "perform" what I want it to do....
2. Make as little as possible changes to the stock/default Ionic gulpfile (using the before/after hooks)
3. Document the new directories - so that it is easy to bundle/inject the "external" libs into the build/js/css

I want to:-
1. Be able to flexibly add modules, just add a new directory, mention the name, and done. A requirement for the above setup.
2. Use Typescript - so, yes, although "dirty", each "external" library in both jspm_packages and in node_modules, otherwise, our IDEs don't "intellisense".
3. Use jspm plugin to "templatecache" (angular2!!) the htmls into ES2015 variables.

..... Some of my thoughts....

Goal is...
To change is little as possible to the default ionic installs....
To let ionic serve/build/run do their jobs - including the ones I've added above
To enjoy the systemJS world
..... To see the module-diagrams mentioned in that course above (want to see an example?)

Regards! Lars
